x-pack/plugins/security_solution/public/timelines/components/flyout/header/index.tsx

```diff
@@ -328,7 +328,7 @@ const FlyoutHeaderComponent: React.FC<FlyoutHeaderProps> = ({ timelineId }) => {
       </EuiFlexItem>
 
       <EuiFlexItem grow={1}>
-        <TimelineKPIs kpis={kpis} isLoading={loading} />
+        {activeTab === 'query' ? <TimelineKPIs kpis={kpis} isLoading={loading} /> : null}
```

Comment:
If you end up having to push any other changes there's an enum `TimelineTabs.query` that you can use here. Not necessary though